app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -176,7 +176,7 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
                   if (!!e?.value) {
                     value = categories.find((c) => c.id === e.value);
                   }
-                  setFieldValue('categories', value);
+                  setFieldValue('categories', !!value ? [value] : []);
```

Comment:
@jdtoombs This was introduced in your last change.  Not sure if what you did fixed one issue and introduced another?